sympy/solvers/ode/tests/test_systems.py

```diff
@@ -430,10 +427,10 @@ def test_sysode_linear_neq_order1():
           Eq(z(t).diff(t), z(t) + Rational(-1, 8) * w(t)),
           Eq(w(t).diff(t), Rational(1, 2) * (w(t) + z(t)))]
 
-    sol6 = [Eq(x(t), 4*C1*exp(3*t/4) + C2*(4*t*exp(3*t/4) + 48*exp(3*t/4)) + C3*exp(t) + C4*t*exp(t)),
-           Eq(y(t), -C1*exp(3*t/4) + C2*(-t*exp(3*t/4) - 8*exp(3*t/4)) + C4*exp(t)),
-           Eq(z(t), C1*exp(3*t/4)/4 + C2*(t*exp(3*t/4)/4 + exp(3*t/4))),
-           Eq(w(t), C1*exp(3*t/4)/2 + C2*t*exp(3*t/4)/2)]
+    sol6 = [Eq(x(t), (C3 + C4*t)*exp(t) + (4*C1 + 4*C2*t + 48*C2)*exp(t)**Rational(3,4)),
+            Eq(y(t), C4*exp(t) + (-C1 - C2*t - 8*C2)*exp(t)**Rational(3,4)),
+            Eq(z(t), (C1/4 + C2*t/4 + C2)*exp(t)**Rational(3,4)),
+            Eq(w(t), (C1/2 + C2*t/2)*exp(t)**Rational(3,4))]
```

Comment:
Can you add a note here linking back to the related issue. We don't really want it to be `exp(t)**Rational(3, 4)`. We should probably find a way to convert that.

Reply:
@oscarbenjamin Sure thing, I will add something like, ```NOTE: To be Simplified```

Reply:
When I say link to the issue I mean add a comment with a URL pointing at the issue like:
```
# There is a problem with collect splitting rationals from the exponent:
# https://github.com/sympy/sympy/issues/19149
```
The idea is that someone looking at the code would be able to find out more information. These comments are written for others to read.